h2o-algos/src/main/java/hex/gam/GAMModel.java

```diff
@@ -28,9 +25,12 @@
 import static hex.glm.GLMModel.GLMParameters.MissingValuesHandling;
 
 public class GAMModel extends Model<GAMModel, GAMModel.GAMParameters, GAMModel.GAMModelOutput> {
+  private static final String[] BINOMIAL_CLASS_NAMES = new String[]{"0", "1"};
+  private static String[] _quasiBinomialDomain;
```

Comment:
so if there are many mojo instances in the JVM they are all supposed to share _quasiBinomialDomain?

Reply:
Definitely should not be static.

Reply:
Removed static.  Thanks!